server/sonar-server/src/test/java/org/sonar/server/permission/ws/AddGroupActionTest.java

```diff
@@ -226,7 +245,8 @@ public void fail_when_project_uuid_and_project_key_are_provided() throws Excepti
   @Test
   public void adding_global_permission_fails_if_not_administrator_of_organization() throws Exception {
     GroupDto group = db.users().insertGroup(defaultOrganizationProvider.getDto(), "sonar-administrators");
-    userSession.login();
+    // user is administrator of another organization
+    userSession.login().addOrganizationPermission("anotherOrg", SYSTEM_ADMIN);
```

Comment:
you could use an override of method `loginAsOrganizationAdmin` here, you could then remove the comment